### ISSUE TITLE
Hue and Alpha slider not working

### ISSUE THREAD

**Issue by Fiju** [casesandberg/react-color#696]
The `Hue` and `Alpha` sliders are not working. 
For `Hue` slider whenever a mouse click event is made it throws the following error:
<img width="1400" alt="Screen Shot 2020-03-31 at 12 05 18 AM" src="https://user-images.githubusercontent.com/29246100/77951658-d7b00080-72e3-11ea-8cbe-d91f02404ef8.png">

And for `Alpha` slider the the error is:
<img width="1398" alt="Screen Shot 2020-03-31 at 12 05 01 AM" src="https://user-images.githubusercontent.com/29246100/77951716-eac2d080-72e3-11ea-86de-3f6e9f980626.png">

I have tried using `SketchPicker`, `ChromePicker` and `SliderPicker`. The errors are same for all the cases.  The error persisted with basic implementation:
```
import { SketchPicker } from 'react-color'
.
.
.
<SketchPicker />
```

**Version:** `2.18.0`

**Comment by bechandrakant** [comment]
I am also facing the same issue.

**Comment by casesandberg** [comment]
This has been fixed and will be release in `react-color@2.18.1`
